packages/settings-library/src/settings_library/director_v0.py

```diff
@@ -0,0 +1,25 @@
+from functools import cached_property
+
+from pydantic import AnyHttpUrl, Field, TypeAdapter
+from settings_library.base import BaseCustomSettings
+from settings_library.basic_types import PortInt, VersionTag
+
+
+class DirectorV0Settings(BaseCustomSettings):
+    DIRECTOR_V0_ENABLED: bool = True
+
+    DIRECTOR_HOST: str = "director"
```

Comment:
why these do not contain `V0` and the other field names do?

Reply:
This is a very old env var which we have been using since the beginning. And the ones without the V0 are also used in the ops env vars.
I will remove the `V0` form here, this will:
- make the naming similar
- allow us to not edit the ops env vars